tests/test_database.py

```diff
@@ -83,7 +82,7 @@ def test_simple(self):
         with open (queryOutputLocation, "r") as myfile:
 
             result = myfile.read()
-            expected = "long.query\t33\tlong\t33\t100.00\t33\n"
+            expected = "long.query\t33\tlong\t33\t100.000\t33"
```

Comment:
Why do you end up with an extra `0` after the decimal here in the test (not that it makes a difference)?

Reply:
I think what happened is the newer version of BLAST reports 3 decimals by default now instead of 2 as before.